### PR TITLE
About page created

### PR DESCRIPTION
**Pull request by arigakova**
Initial version of About page was successfully created.

### CODE REVIEW

File: src/images/photoDog.png
Comment (badgerbadgers) [inline src/images/photoDog.png:1]: This is a great pic!

File: src/component/layout/About/About.css
Comment (badgerbadgers) [inline src/component/layout/About/About.css:1]: Might be worth deciding next team meeting any frameworks or libraries you will use like Bootstrap, tailwind css or Material UI and how you as a group will style the application, (will each component have a css file? one global css file that will style your html elements like h1,h2, will you use CSS modules?)

**Review by badgerbadgers — APPROVED** [review]
LGTM! App is looking good.

**Comment by AlonaVladymyrovaTrinity** [comment]
Looks good for me!

**Review by AlonaVladymyrovaTrinity — APPROVED** [review]

**Review by LiubovCass — APPROVED** [review]
Looks good! No comments from my side

**Pull request merged**